alembic/testing/util.py

```diff
@@ -78,3 +80,11 @@ def run_ddl(self):
         return fixture_functions.fixture(scope=ddl)(run_ddl)
 
     return decorate
+
+
+def path(linuxPath):
```

Comment:
oops no camels allowed :)

flake8 will probably hit this

Reply:
I usually code in camels so sometimes I use them out of habit